python/rds/app.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+from aws_cdk import (
+    aws_ec2 as ec2,
+    aws_rds as rds,
+    core
+)
+
+
+class RDSStack(core.Stack):
+    def __init__(self, app: core.App, id: str, **kwargs) -> None:
+        super().__init__(app, id, **kwargs)
+
+        vpc = ec2.Vpc(self, "VPC")
+        
+        rds.DatabaseInstance(
+            self, "RDS",
+            master_username="master",
+            master_user_password=core.SecretValue("password"),
+            database_name="db1",
+            engine_version="8.0.16",
+            engine=rds.DatabaseInstanceEngine.MYSQL,
+            vpc=vpc,
+            port=3306,
+            instance_class=ec2.InstanceType.of(
+                ec2.InstanceClass.MEMORY4, 
+                ec2.InstanceSize.LARGE
+            ),
+            removal_policy=core.RemovalPolicy.DESTROY,
```

Comment:
Could you also add `deletion_protection=False` here? I just tried to deploy the stack, and now I can't delete it because of that. Feel free to add a comment that you probably want to change these 2 properties (`removal_policy` and `deletion_protection`) to their defaults if you're creating a production RDS database.